subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -222,7 +236,8 @@ static int job_update_accepted(struct mqtt_client *const client,
 	 */
 	execution_version_number++;
 
-	if (fota_state == DOWNLOAD_FIRMWARE) {
+	if (fota_state == DOWNLOAD_FIRMWARE
```

Comment:
Could you reorder this so that the check is done the same as the `else if` below?

Reply:
Done. I think maybe we could simplify the `if` by re-arranging it. You would loose a bit readability since the control flow won't be linear.